rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ProjectSpecTest.java

```diff
@@ -483,4 +492,14 @@ public void should_document_create_project() throws Exception {
                                 fieldWithPath("_embedded.createdBy").description("The user who created this project")
                         )));
     }
+
+	@Test
+	public void should_document_create_releases() throws Exception {
```

Comment:
I know, that the method in the rest controller is also called 'create releases', but I find that confusing since no one creates releases. A release is linked, but not created. Anyway, since the controller uses this strange naming, we can leave it here I think (or we should change both, the controller and the documentation here).

Reply:
Updated the method names in both the places.